### ISSUE TITLE
Custom time states not properly initialized

### ISSUE THREAD

**Issue by ZachBenz** [ZachBenz/RingForIndigo#50]
I failed to account for previously existing states still set to the initialization default of January 1, 1900.  Thus, when we try to initialize the localized time, if we try to convert the UTC time of 1/1/1900, for anyone east of UTC, this results in a localized time in 1899, which is not valid.

Fortunately, this is largely a cosmetic issue, so everything should continue to function, just a couple of device states won't be properly initialized to default values.

**Comment by ZachBenz** [comment]
Perhaps just catch exception:

`Ring Error                      exception in deviceStartComm(Front Doorbell): year=1899 is before 1900; the datetime strftime() methods require year >= 1900`

And use distantPast to initialize

Alternative is to realize that date is 1/1/1900 and won't localize properly... but this requires more complicated logic prone to coding error.

**Comment by ZachBenz** [comment]
Pushed out a fix in version 1.7.5 - awaiting feedback from users in support forum on if issue resolved.

**Comment by ZachBenz** [comment]
Per user feedback in support forum, 1.7.5 did not fix:

```
Jan 13, 2020 at 8:15:32 AM
   Stopping plugin Ring (pid 6890)
   Stopped plugin Ring
   Upgrading plugin Ring to newer version 1.7.5 (previous version moved to trash)
   Loading plugin "Ring 1.7.5"
   Starting plugin "Ring 1.7.5" (pid 11381)
   Started plugin "Ring 1.7.5"
   Ring Error                      exception in deviceStartComm(Front Doorbell): year=1899 is before 1900; the datetime strftime() methods require year >= 1900
   Ring                            Attempting to connect to Ring.com API and login as ryanbuckner@gmail.com
   Ring                            Connection to Ring.com API successful
```

The code triggering the exception is earlier in the chain then where the try statement currently is placed - need to fix this.
